proxy/src/main/java/net/md_5/bungee/connection/BungeePluginChannelHandler.java

```diff
@@ -126,6 +126,12 @@ private void handleMessage(DataInput in, ByteArrayDataOutput out) throws Excepti
         if ( target == null ) { return; }
         target.sendMessage( new TextComponent( in.readUTF() ) );
     }
+    
+    private void handleRawMessage(DataInput in, ByteArrayDataOutput out) throws Exception {
```

Comment:
Any chance I could get you to move this down under handlePlayerList so the functions remain in properly alphabetical order? Makes it easier to find what you're looking for in the file. (I'm a bit OCD about this, LOL)